src/main/java/racing/Car.java

```diff
@@ -0,0 +1,30 @@
+package racing;
+
+/**
+ * @author han
+ */
+public class Car {
+    private int step = 0;
+
+    public Car() {
+    }
+
+    public Car(int step) {
+        this.step = step;
+    }
+
+    public int getStep() {
+        return this.step;
+    }
+
+    public Car getInstanceByForward(int step) {
+        if (isForward(step)) {
+            this.step = Math.addExact(this.step, 1);
+        }
+        return new Car(this.step);
+    }
+
+    private boolean isForward(int step) {
+        return step > 4;
```

Comment:
요구사항 중 자동차의 전진 조건은 4 `이상` 입니다.
매직넘버를 치환하여 의미를 부여하면 어떨까요?
```suggestion
        return step >= FORWARD_STANDARD;
```